entrypoint.sh

```diff
@@ -38,7 +38,11 @@ fi
 # This possibly changes the log directory that might be mounted in
 if check_set "${ENABLE_AIRFLOW_CHOWN}"; then
   echo "Chowning ${AIRFLOW_HOME} to ${AIRFLOW_USER}:${AIRFLOW_GROUP}..."
-  chown "${AIRFLOW_USER}:${AIRFLOW_GROUP}" -R "${AIRFLOW_HOME}/"
+  mkdir -pv "${AIRFLOW_HOME}/dags"; \
+  mkdir -pv "${AIRFLOW_HOME}/logs"; \
```

Comment:
Before `entrypoint.sh` is run, the directory looks like:

```
root@308c5db7057c:/airflow# ls -la
total 664
drwxr-xr-x 1 root root   4096 Jun 10 22:53 .
drwxr-xr-x 1 root root   4096 Jun 10 22:52 ..
-rw-r--r-- 1 root root  42162 Jun 10 22:53 airflow.cfg
-rw-r--r-- 1 root root 598016 Jun 10 22:53 airflow.db
drwxr-xr-x 2 root root   4096 Jun 10 14:37 dags
drwxr-xr-x 3 root root   4096 Jun 10 22:53 logs
-rw-r--r-- 1 root root   2129 May 24 07:52 setup_auth.py
-rw-r--r-- 1 root root   2198 Jun  7 05:34 test_db_conn.py
-rw-r--r-- 1 root root   4700 Jun 10 22:53 webserver_config.py
```

`airflow db init` will create the `/logs` file, so i assume this is a change from V2 onwards.

Either create the `logs` directory first and chown it or run `airflow db init` then chown the created `logs` directory. I prefer the former.